src/styles/styles.js

```diff
@@ -1580,6 +1580,33 @@ const styles = {
         height: 80,
     },
 
+    chatAvatar: {
```

Comment:
I think this name might be too overly generic, given that we have "chat avatars" that show up all over the app. Perhaps something like "emptyStateAvatar" is better? Curious for your thoughts @roryabraham

Reply:
Depending on how we name this, that might affect the names you are using in subsequent lines.

Reply:
i had changed ChatAvatars File and style name to emptyStateAvatar.